model/DataStore/MetaDataDeliverySyncTask.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoDeliveryRdf\model\DataStore;
+
+use common_exception_Error;
+use common_exception_NotFound;
+use JsonSerializable;
+use oat\oatbox\extension\AbstractAction;
+use oat\oatbox\filesystem\FileSystem;
+use oat\oatbox\filesystem\FileSystemService;
+use oat\oatbox\reporting\Report;
+use oat\oatbox\service\ConfigurableService;
+use oat\oatbox\service\exception\InvalidServiceManagerException;
+use oat\tao\helpers\FileHelperService;
+use oat\tao\model\taskQueue\QueueDispatcher;
+use taoQtiTest_models_classes_export_TestExport22;
+use Throwable;
+
+class MetaDataDeliverySyncTask extends AbstractAction implements JsonSerializable
+{
+    private const MAX_TRIES = 1;
```

Comment:
This will probably come from a `DeliveryMetadataListener` injected in the Task payload ($params).